packages/@react-types/list/src/index.d.ts

```diff
@@ -11,4 +11,5 @@
  */
 
 export type {AriaGridListProps, GridListProps} from '@react-aria/gridlist';
+export type {ListState} from '@react-stately/list';
```

Comment:
The exports here for `AriaGridListProps`, `GridListProps`, and `SpectrumListViewProps` were kept in here to preserve back compat, but these props are now exported from their specific packages. IMO we can go ahead and just export ListState from `@react-stately/list` if needed, where did you need this type specifically?

Reply:
Good point, I went ahead and removed this export